composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "proximax/php-xpx-chain-sdk",
-    "description": "Composer Package for PHP, Wrapper to use the NEM blockchain REST API and SDK with NIS compatibility.",
-    "keywords": ["nem", "cryptocurrency", "api", "sdk", "restful", "nem-php", "php-nem"],
+    "description": "Composer Package for PHP, Wrapper to use the Proximax blockchain REST API and SDK with NIS compatibility.",
```

Comment:
Wrapper to use the ProximaX Sirius-Chain REST API and SDK. (remove the `with NIS compatibility)`